django_react_proj/users-fe/src/components/RealTime/RealTime.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import moment from 'moment'
```

Comment:
Moment.js is a deprecated library not really in maintenance. It's also really heavy compared to other libraries (and your needs) It's better to use `date-fns` or `dayjs`.

Reply:
Changed to date-fns